packages/shared/is-visible.js

```diff
@@ -5,13 +5,20 @@
  */
 
 function isStyleVisible(element) {
-  const { display, visibility, opacity } = element.style
+  if (!(element instanceof HTMLElement) && !(element instanceof SVGElement)) {
+    return false
+  }
+
+  // Per https://lists.w3.org/Archives/Public/www-style/2018May/0031.html
+  // getComputedStyle should only work with connected elements.
+  const { display, visibility, opacity } = element.isConnected
+    ? getComputedStyle(element)
+    : element.style
   return (
     display !== 'none' &&
     visibility !== 'hidden' &&
     visibility !== 'collapse' &&
-    opacity !== '0' &&
-    opacity !== 0
```

Comment:
We don't need this last check anymore? The `!== 0`?

Reply:
I removed it since you removed it in v2 (https://github.com/vuejs/test-utils/commit/9f343b049a815c9832445067b9a419fbc448d428), but can easily add it back if you'd feel more comfortable doing that? It appears to always be a string value, but can't say 100% sure.

Reply:
Hi, I'm sorry about the slow reply, I was away from computer/internet during Oct, but I am back now.

I agree, let's get this merged.

Reply:
🎊  No problem, thanks a lot!

Reply:
It's out, grab `@vue/test-utils@1.3.2` or `@vue/test-utils@legacy`.

Reply:
https://github.com/vuejs/vue-test-utils/releases/tag/v1.3.2